docs/how-to/configuring-validating-environment.rst

```diff
@@ -0,0 +1,71 @@
+.. meta::
+   :description: Omnitrace documentation and reference
+   :keywords: Omnitrace, ROCm, profiler, tracking, visualization, tool, Instinct, accelerator, AMD
+
+****************************************************
+Configuring and validating the environment
+****************************************************
+
+After installing the `Omnitrace <https://github.com/ROCm/omnitrace>`_ application, some additional steps are required to set up
```

Comment:
Should this be:
After installing OT, additional steps are...

Not sure what the OT application is